docs/sdks/sdk-ref-python.md

```diff
@@ -70,6 +70,7 @@ The initialization step depends on the role, as shown in the following table.
 | Role                                      | Create Instance of Class | Link to Instructions                                                                  |
 |:------------------------------------------|:-------------------------|:--------------------------------------------------------------------------------------|
 | Publisher                                 | `Uid2PublisherClient`    | [Usage for Publishers](#usage-for-publishers)                                         |
+| Advertisers and Data Providers            | `IdentityMapClient` | [Usage for Advertisers and Data Providers](#usage-for-advertisers-and-data-providers)     |
```

Comment:
Other roles in the table are singular (publisher, DSP, Sharer) so this role should be also (Advertiser/Data Provider).
Note: If you copied this from somewhere else LMK and we can make them consistent. Either is really fine but we have to be consistent.